### PR TITLE
Bug 1288404: don't expect a specific schedulerId for tasks

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

File: src/treeherder/action_handler.js [hunk 1]
```diff
@@ -42,8 +42,9 @@ class Handler {
   }
 
   async handleRetrigger(taskId, runId, task, payload) {
-    // Must be a scheduled task and have a task group ID
-    if (task.schedulerId !== SCHEDULER_TYPE || !task.taskGroupId) {
+    // Must have a task group ID
+    if (!task.taskGroupId) {
```
Comment (jonasfj) [inline src/treeherder/action_handler.js]: If loaded from the queue it **always** has a `taskGroupId`. It's a required property, see docs...

(not required for creation, but always given when returned, it defaults to the taskId if a taskGroupId isn't given).

**Comment by jonasfj** [comment]
# `r+`

I don't understand the entire context here, but the changes looks okay... Pretty much you just ignore the `schedulerId` which is fine...

**Comment by jonasfj** [comment]
@djmitche, note `schedulerId` is now meaning the person/entity that scheduled something... So maybe it should be `try-push` for try tasks... or something like that... But I guess that could be a different story for another day.

**Comment by djmitche** [comment]
Yeah, I wasn't sure what that taskGroupId check was for.. I'll remove it.

Yes, naming schedulers is an ongoing project (greg will be working on it when he's back.. #86 is a start on it)

File: src/treeherder/action_handler.js [hunk 2]
```diff
@@ -58,7 +53,7 @@ class Handler {
     let parsedRoute = parseRoute(route);
 
     if (!parsedRoute.revision && !parsedRoute.revisionHash) {
-      console.log(`Could not determine revision hash from task ${taskId}.  Not retriggering.`);
+      console.log(`Could not determine revision hash while retriggering task ${taskId}.  Not retriggering.`);
```
Comment (djmitche) [inline src/treeherder/action_handler.js]: This is just so a search for "retrigger" will show this error, btw

**Pull request merged**